sdk/src/impl/position-impl.ts

```diff
@@ -224,7 +229,7 @@ export class PositionImpl implements Position {
 
   async collectFees(
     updateFeesAndRewards: boolean = true,
-    resolveATA: boolean = true,
```

Comment:
i think we should pass in resolveATA:boolean instead.
When you are composing this fn, you are resolving ATA outside and you can use false here. The method will still derive the aTA but will not resolve it.